source/GameInterface/Services/MobileParties/Patches/WageChangesSettlementPatch.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Common;
+using Common.Messaging;
+using Common.Util;
+using GameInterface.Services.MobileParties.Messages;
+using HarmonyLib;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Threading.Tasks;
+using TaleWorlds.CampaignSystem.Party;
+using TaleWorlds.CampaignSystem.ViewModelCollection.ClanManagement;
+
+namespace GameInterface.Services.MobileParties.Patches;
+
+
+/// <summary>
+/// Used to allow a party to modify wages
+/// </summary>
+[HarmonyPatch(typeof(ClanFinanceExpenseItemVM))]
+internal class WageChangesSettlementPatch
+{
+    private static MethodInfo MobileParty_SetWagePaymentLimit = typeof(MobileParty).GetMethod(nameof(MobileParty.SetWagePaymentLimit));
+
+    private static MethodInfo MobileParty_SetWagePaymentLimitOverride =
+        typeof(WageChangesSettlementPatch).GetMethod(nameof(SetWagePaymentLimitOverride), BindingFlags.NonPublic | BindingFlags.Static);
+
+
+    [HarmonyPatch("OnUnlimitedWageToggled")]
+    [HarmonyPatch("OnCurrentWageLimitUpdated")]
```

Comment:
Having 2 functions here does not work afaik. You can use [TargetMethods](https://harmony.pardeike.net/articles/patching-auxiliary.html#targetmethods) instead for mutiple